Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppNode.java

```diff
@@ -48,15 +51,18 @@
         + "Are you sure you want to continue?";
     private static final String DELETE_WEBAPP_PROGRESS_MESSAGE = "Deleting Web App";
     private static final String LABEL = "WebApp";
+    public static final String SSH_INTO = "SSH into Web App";
```

Comment:
private

Reply:
it will be used in a external class as a constant varible.